tools/contractdeployer/cli_flags.go

```diff
@@ -2,18 +2,18 @@ package contractdeployer
 
 // Flag names, defaults and usages.
 const (
-	l1NodeHostName  = "l1NodeHost"
-	l1NodeHostUsage = "The host on which to connect to the Ethereum client"
+	nodeHostName  = "nodeHost"
+	nodeHostUsage = "The host on which to connect the RPC client"
 
-	l1NodePortName  = "l1NodePort"
-	l1NodePortUsage = "The port on which to connect to the Ethereum client"
+	nodePortName  = "nodePort"
+	nodePortUsage = "The port on which to connect the RPC client"
 
-	mgmtContractPathName  = "mgmtContractPath"
-	mgmtContractPathUsage = "The path of the management contract path"
+	contractName  = "contract"
```

Comment:
Sugestion: Would change this flag to `contractID` or `contractName`, maybe something else ? Just to make clear that it's and identifier of sorts ? WDYT ?

Reply:
Nice, yeah good idea, makes it clearer what the argument expects (and it will be fun to call this variable `contractNameName` haha